lib/child-writer/index.js

```diff
@@ -126,8 +126,22 @@ exports.write = (image, drive, options) => {
    * }));
    */
   const bridgeRobotMessage = (message) => {
+    const ROBOT_COMMAND_LOG = 'log';
+
     try {
-      const parsedMessage = robot.parseMessage(message);
+      const parsedMessage = _.attempt(() => {
```

Comment:
The point of this whole try/catch block is to emit the error as an event instead of just throwing. Even though we used `_.attempt`, we still want to emit any error that could come from it.